zh/deploy-multicluster-tidb-cluster.md

```diff
@@ -0,0 +1,377 @@
+---
+title: 跨多个 Kubernetes 集群部署 TiDB 集群
+summary: 本文档介绍如何实现跨多个 Kubernetes 集群部署 TiDB 集群
+---
+
+## 什么是跨多个 Kubernetes 集群部署 TiDB 集群
+
+跨多个 Kubernetes 集群部署 TiDB 集群是指在多个互相联通的 Kubernetes 集群部署数据联通, 可以在集群间容灾, 扩缩容的 TiDB 集群. 所谓 Kubernetes 集群互相连通, 在此文档中指部署的目标 Kubernetes 集群都处于同一网络环境下(如同一 VPC 下), POD IP 在任意集群内和集群间可互相访问, FQDN 记录是集群内和集群间均可被解析的. 满足上述条件的 Kubernetes 集群可以进行 TiDB 集群部署.
+
+## 前置条件
+
+- 各集群上的 POD IP 可以在同一网络/VPC 内被互相访问
+- 集群上使用了 Cluster Domain 的 FQDN 记录可以在集群间解析和访问
+
+## 目前支持场景
+
+- 部署新的开启跨多个 Kubernetes 集群的 TiDB 集群, 允许在其他 Kubernetes 集群上部署开启此功能的新集群加入同样开启此功能的集群
+
+## 实验性支持场景
+
+- 对已有数据的集群从未开启此功能状态变为开启此功能状态, 生产使用建议通过数据迁移完成此需求
+
+## 不支持场景
+
+- 两个已有数据集群互相连通, 对于这一场景应通过数据迁移完成
```

Comment:
This should be 3 paragraphs put in the first section, right below L9.

Reply:
L10 is closely related to the introduction part. It's better not to break them.